packages/streamline/src/primitives/icon/icon.constants.ts

```diff
@@ -2,10 +2,12 @@ export const ICON_SIZE = {
   small: 12,
   regular: 16,
   large: 24,
+  xlarge: 48,
 };
 
 export const SPINNER_SIZE = {
   small: 16,
   regular: 24,
   large: 48,
+  xlarge: 48,
```

Comment:
Discussed with @hcourthias , we may change it later but for now, it's just to match the icon different possibilities but should be of no use.